src/stirling/source_connectors/socket_tracer/protocols/common/data_stream_buffer.h

```diff
@@ -147,7 +161,10 @@ class DataStreamBuffer {
   void ShrinkToFit() { impl_->ShrinkToFit(); }
 
  private:
+  StatusOr<uint64_t> GetPrevTimestamp() const { return prev_timestamp_ns_; }
+  void SetPrevTimestamp(const StatusOr<uint64_t>& timestamp) { prev_timestamp_ns_ = timestamp; }
   std::unique_ptr<DataStreamBufferImpl> impl_;
+  StatusOr<uint64_t> prev_timestamp_ns_;
```

Comment:
Did you consider storing the `uint64_t` directly? I have a slight preference to unwrap it and keep the integer value rather than the `StatusOr` object.

Reply:
Changed to `uint64_t`